internal/core/src/segcore/ChunkedSegmentSealedImpl.cpp

```diff
@@ -1766,14 +1774,11 @@ ChunkedSegmentSealedImpl::search_ids(const IdArray& id_array,
     auto res_id_arr = std::make_unique<IdArray>();
```

Comment:
res_id_arr is never used by the only caller of this function, we may consider remove it. not requiring a change though.